### ISSUE TITLE
Train rq-vae on FFHQ failed when using the  default ffhq256-rqvae-8x8x4.yaml

### ISSUE THREAD

**Issue by xiapengchng** [kakaobrain/rq-vae-transformer#23]
I trt to retrain rq-vae on FFHQ with the  default ffhq256-rqvae-8x8x4.yaml. The training loss first decrease and then increase.
![image](https://user-images.githubusercontent.com/32598987/223308206-0dc2cab7-9583-48c2-a229-16041b544f6e.png)

and Then I compute rfid using the model with the lowest valid loss epoch=65 valid total loss=1.1. the rfid=317.
![image](https://user-images.githubusercontent.com/32598987/223308459-7990a39c-ad6c-45b9-993b-2944ef13c028.png)

Is there anyting wrong?

**Comment by chiheonk** [comment]
Hi @xiapengchng,
Can you provide more details, including
- code or command line script you ran, and
- number of gpus and torch version?

**Comment by drx-code** [comment]
Hi @xiapengchng, did you solve that problem?
